mlflow/tracing/processor/mlflow.py

```diff
@@ -122,6 +123,8 @@ def _start_trace(self, span: OTelSpan) -> TraceInfo:
                     "backend APIs are not available. Fallback to client-side generation",
                     exc_info=True,
                 )
+                if _MLFLOW_TESTING.get():
+                    raise
```

Comment:
Can this be removed?

Reply:
Yea I think this helps for debugging/testing, but I'll remove